src/concat_bar_layer_circle.vl.js

```diff
@@ -5,71 +5,50 @@ import * as vl from 'vega-lite-api';
  * generate and return the vega-lite JSON spec below.
  */
 
-/*
-{
-  "$schema": "https://vega.github.io/schema/vega-lite/v5.json",
-  "description": "A dashboard with cross-highlighting.",
-  "data": {"url": "data/movies.json"},
-  "vconcat": [
-    {
-      "layer": [{
-        "mark": "circle",
-        "encoding": {
-          "x": {
-            "bin": {"maxbins": 10},
-            "field": "IMDB Rating"
-          },
-          "y": {
-            "bin": {"maxbins": 10},
-            "field": "Rotten Tomatoes Rating"
-          },
-          "size": {
-            "aggregate": "count",
-            "title": "All Movies Count"
-          },
-          "opacity": {"value": 0.4}
-        }
-      }, {
-        "transform": [{
-          "filter": {"param": "pts"}
-        }],
-        "mark": "circle",
-        "encoding": {
-          "x": {
-            "bin": {"maxbins": 10},
-            "field": "IMDB Rating"
-          },
-          "y": {
-            "bin": {"maxbins": 10},
-            "field": "Rotten Tomatoes Rating"
-          },
-          "size": {
-            "aggregate": "count",
-            "title": "Selected Category Count"
-          }
-        }
-      }],
-      "resolve": {"legend": {"size": "independent"}}
-    }, {
-      "width": 330,
-      "height": 120,
-      "mark": "bar",
-      "params": [{
-        "name": "pts",
-        "select": {"type": "point", "encodings": ["x"]}
-      }],
-      "encoding": {
-        "x": {"field": "Major Genre", "axis": {"labelAngle": -40}},
-        "y": {"aggregate": "count"},
-        "color": {
-          "condition": {
-            "param": "pts",
-            "value": "steelblue"
-          },
-          "value": "grey"
-        }
-      }
-    }
-  ]
+
+const vl = require('vega-lite-api');
+
+function generateVegaLiteSpec() {
```

Comment:
Thanks for the contribution. Take a look at the structure of some of the already completed ones and you'll see you can save a few steps. Change this to `export default function chart() {` (the name doesn't matter but may help to keep it consistent